src/app/app.material.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import {MatButtonModule, MatCheckboxModule} from '@angular/material';
+
+@NgModule({
+  imports: [MatButtonModule, MatCheckboxModule],
```

Comment:
I started to create an array of Material modules and then reference that array in imports and exports, so that you only need to add the module once, what say?

Reply:
Most of the docs and tutorials say this .. but the code gets messy when you have a lot of modules

Reply:
Something like this: https://stackoverflow.com/a/49294077

Reply:
good idea. pushed